<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>doctorAdviceClient</class>
 <widget class="QWidget" name="doctorAdviceClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>568</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>879</width>
    <height>568</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>879</width>
    <height>568</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>医嘱客户端</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* 主窗口样式 */
QWidget {
    background-color: #f8f9fa;
    font-family: 'Segoe UI', 'Microsoft YaHei', sans-serif;
}

/* 按钮样式 */
QPushButton {
    background-color: #ffffff;
    border: 1px solid #e0e0e0;
    border-radius: 6px;
    padding: 8px 16px;
    color: #424242;
    font-weight: 500;
    font-size: 14px;
    transition: all 0.2s ease;
}

QPushButton:hover {
    background-color: #f5f5f5;
    border: 1px solid #bdbdbd;
    color: #212121;
}

QPushButton:pressed {
    background-color: #eeeeee;
    border: 1px solid #9e9e9e;
}

/* 输入框样式 */
QLineEdit {
    background-color: #ffffff;
    border: 1px solid #e0e0e0;
    border-radius: 4px;
    padding: 8px 12px;
    color: #424242;
    font-size: 16px;
    selection-background-color: #e3f2fd;
}

QLineEdit:focus {
    border: 1px solid #64b5f6;
    outline: none;
}

QLineEdit:hover {
    border: 1px solid #bdbdbd;
}

/* 标签样式 */
QLabel {
    color: #424242;
    font-size: 16px;
    font-weight: 500;
    padding: 8px 0px;
    background-color: transparent;
}</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #f5f5f5;
    border: 1px solid #e0e0e0;
}

QPushButton:hover {
    background-color: #e0e0e0;
}</string>
   </property>
   <property name="text">
    <string>编辑</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>160</y>
     <width>541</width>
     <height>321</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="medicineLineEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>请输入药品信息</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="checkLineEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>请输入检查信息</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="therapyLineEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>请输入治疗信息</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="careLineEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>请输入护理信息</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>60</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #42a5f5;
    color: white;
    border: 1px solid #1e88e5;
}

QPushButton:hover {
    background-color: #2196f3;
    border: 1px solid #1976d2;
}

QPushButton:pressed {
    background-color: #1e88e5;
}</string>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>151</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>-1</pointsize>
        <fontweight>Medium</fontweight>
       </font>
      </property>
      <property name="text">
       <string>药品</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>-1</pointsize>
        <fontweight>Medium</fontweight>
       </font>
      </property>
      <property name="text">
       <string>检查</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>-1</pointsize>
        <fontweight>Medium</fontweight>
       </font>
      </property>
      <property name="text">
       <string>治疗</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>-1</pointsize>
        <fontweight>Medium</fontweight>
       </font>
      </property>
      <property name="text">
       <string>护理</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
